Makefile

```diff
@@ -36,10 +36,17 @@ TESTFLAGS    :=
 DUPLFLAGS    := -t 100
 
 ifeq ($(STATIC),1)
+# Static linking with glibc is a bad time. If we're building statically,
+# assume musl is installed and use that.
```

Comment:
I'd prefer a separate script (probably not even in the repo; maybe in a wiki page or something). There are two pieces of hard-earned knowledge here: that musl is a way to make fully-static linux builds (along with the other flags used here), and that you probably don't need a static build even if you think you might. The latter part is more important so it's more reliably conveyed in a format that is not immediately executable.

> By the way, isn't hostname resolution etc unaffected since we're using netgo in static builds?

Good point. So static builds are incompatible with regular builds whether we use musl or not.